TempoWorld/Source/TempoWorld/Private/TempoActorControlServiceSubsystem.cpp

```diff
@@ -945,7 +1003,8 @@ grpc::Status SetPropertyImpl<SetFloatPropertyRequest>(const UWorld* World, const
 {
 	// First try to set it as a double, then fall back on float
 	const grpc::Status FloatStatus = SetSinglePropertyImpl<FDoubleProperty>(World, Request, Request.value());
-	if (FloatStatus.ok())
+	// If we got an error other than FAILED_PRECONDITION that means the type was right, but something else was wrong.
+	if (FloatStatus.ok() || FloatStatus.error_code() != grpc::FAILED_PRECONDITION)
```

Comment:
This is an unrelated fix. If the property was right but we got another error, we should quit and report that error.